src/renderer/viz/expressions/ramp.js

```diff
@@ -455,3 +454,16 @@ function _calcPaletteValues (palette) {
 
     return palette;
 }
+
+function _calcColorValueIndex (texturePixels, m) {
+    switch (m) {
+        case Number.isNaN(m):
+            return 0;
+        case m === Infinity:
+            return texturePixels.length - 4;
```

Comment:
I think this is buggy, see line `r: Math.round(texturePixels[index * 4 + 0]),` the index is going to be multiplied by 4 and I think it will overflow

Reply:
OPS you are right! let me change it